backend/CHANGELOG.md

```
@@ -1,4 +1,8 @@
 # Changelog
+## [3.1.7] - 2025-04-16
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)